### DISCUSSION TITLE
Zag.js + Mitosis

### DISCUSSION THREAD

**Discussion by coopbri** [BuilderIO/jsx-lite#1349] [Q&A]
Hello, I noticed the README hints at examples combining Mitosis and Zag.js:

> While Zag.js focuses on writing framework-agnostic interactions, Mitosis focuses on writing framework-specific components. (PS: We are working on building examples that combine these two together. TBD!)"

I am maintaining a design system built on Zag.js, Ark UI, and Panda CSS, and would love to figure out how to integrate these! Currently focusing on React components in my DS but it would be so nice to have per-framework components "compiled" via Mitosis.

My question is: are there any WIP/draft examples of this integration I can use as reference somewhere?

Somewhat similar discussion (Mitosis + Panda CSS): https://github.com/BuilderIO/mitosis/discussions/1331

**EDIT:** there is some discussion about this on the Ark UI repo [here](https://github.com/chakra-ui/ark/discussions/1819#discussioncomment-7814785)

**Comment by juliomuhlbauer** [comment]
It would be amazing to see Ark UI using Mitosis. Because the Ark UI codebase is simply a lot of framework rewriting of Zag. It would be a lot easier to maintain de codebase.
